### ISSUE TITLE
Update to OwnCloud 8.1.1 with enabled lucene full text search app fails

### ISSUE THREAD

**Issue by motlib** [owncloud/core#18327]
Hello, 

I tried to upgrade OwnCloud to version 8.1.1. During update the following error message is shown and the upgrade fails:

```
Doctrine\DBAL\Exception\TableNotFoundException: An exception occurred while executing 'DELETE FROM `oc_queuedtasks` WHERE `app`=?' with params ["search_lucene"]: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'owncloud.oc_queuedtasks' doesn't exist
```

I have installed the lucene full text index app from the app store. So I think the problem is caused by that app. I can try the upgrade again, but it fails every time again with the same message.

Thanks for any help or hint to get the OwnCloud instance running again.

Motlib

**Comment by MorrisJobke** [comment]
If the file `apps/search_lucene/appinfo/update.php` looks like this:

https://github.com/owncloud/search_lucene/blob/a8ed3775ddcd0868ff7674906e923252e2dd8b6f/appinfo/update.php

You simply can delete this file and retry the upgrade.

**Comment by RealRancor** [comment]
Dup of https://github.com/owncloud/search_lucene/issues/90 and https://github.com/owncloud/core/issues/18247

**Comment by MorrisJobke** [comment]
Fix is in owncloud/search_lucene#101

**Comment by MorrisJobke** [comment]
backport request to stable8.1 is pending.

**Comment by MorrisJobke** [comment]
Backport owncloud/search_lucene#102

**Comment by MorrisJobke** [comment]
This will be fixed in 8.1.2
